### DISCUSSION TITLE
How to integrate Shuffle with IRIS

### DISCUSSION THREAD

**Discussion by FarisThabit** [Shuffle/Shuffle#1455] [Q&A]
I am using the Shuffle installed on docker, and I want to integrate Shuffle with IRIS. How can I do that?

![Shuffle and IRIS](https://github.com/user-attachments/assets/c7eed4c8-24cc-4fd3-bd64-57882f58e793)

**Comment by ArfanAbid** [comment]
Hey, Check this out:
https://github.com/ArfanAbid/SOC-Stack-Workflows

**Comment by ggambini** [comment]
Hello

I use iris webhook module to call a workflow on shuffle side.

Regards
